docs/getting-started/integrations/angular.md

```diff
@@ -171,168 +190,53 @@ Finally, use the `<ckeditor>` tag in the template to run the rich text editor:
-      element
-    );
-  }
-}
-```
-
-And then, in the template:
-
-```html
-<!-- app.component.html -->
-
-<ckeditor [editor]="Editor" data="<p>Hello, world!</p>" (ready)="onReady($event)"></ckeditor>
-```
-
-### Using a custom CKEditor&nbsp;5 build
-
-If you want to add more plugins to an existing build or customize something that cannot be controlled with the {@link getting-started/setup/configuration editor configuration} you need to create a custom build first, as described in the {@link getting-started/legacy-getting-started/quick-start-other#building-the-editor-from-source Building the editor from source} guide.
-
-By completing the above tutorial you should get a generated `ckeditor.js` file (and corresponding translation files). In the next step you should copy it to the `src` directory and import it to the component file.
-
-```ts
-// app.component.ts
-
-import * as Editor from 'path/to/the/ckeditor';
-
-@Component( {
-	// ...
-} )
-export class MyComponent {
-	public Editor = Editor;
-	// ...
-}
-```
-
-Note that to allow importing JavaScript files without providing their corresponding types you need to set `allowJs` to `true` in the `tsconfig.json` file. Also, make sure that you target `ES6` or higher, otherwise you are likely to end up with a [weird transpilation error](https://github.com/ckeditor/ckeditor5-angular/issues/20) in the production build.
-
-```json
-// tsconfig.json
-
-"compilerOptions": {
-	"allowJs": true,
-	"target": "es2015"
-	// other options
-}
-```
-
-<info-box>
-	If you cannot set the target higher than `es5`, try to set `"buildOptimizer": false` which will produce a bigger, but correct production build.
-</info-box>
-
-### Integrating a build from the online builder
-
-This guide assumes that you have created a zip archive with the editor built using the [CKEditor&nbsp;5 online builder](https://ckeditor.com/ckeditor-5/online-builder/).
-
-Unpack it into you application's main directory. The directory with the editor build cannot be placed inside the `src/` directory as Node will return an error. Because of that, we recommend placing the directory next to the `src/` and `node_modules/` folders:
-
-```
-├── ckeditor5
-│   ├── build
-│   ├── sample
-│   ├── src
-│   ├── ...
-│   ├── package.json
-│   └── webpack.config.js
-├── node_modules
-├── src
-├── ...
-└── package.json
-```
-
-Then, add the package located in the `ckeditor5` directory as a dependency of your project:
-
-```
-npm install ./ckeditor5
-```
-
-Now, import the build in your application:
-
-```ts
-// app.component.ts
+import { CKEditorModule } from '@ckeditor/ckeditor5-angular';
+import { ClassicEditor, Bold, Essentials, Italic, Mention, Paragraph, Undo } from 'ckeditor5';
+import { SlashCommand } from 'ckeditor5-premium-features';
 
-import { Component } from '@angular/core';
-import Editor from 'ckeditor5-custom-build/build/ckeditor';
+import 'ckeditor5/index.css';
+import 'ckeditor5-premium-features/index.css';
```

Comment:
Just a cautionary comment, as I'm a bit lost, should `dist` be here or not?

Reply:
Isn't `dist` only in HTML calls? Not in JS calls?

Reply:
AFAIK we needed the `dist` prefix in some earlier alpha versions. But, there was a change to the paths. We don't need it for npm, but it's required for the CDN, though.

Reply:
OK!